src/wayland/shell/xdg/mod.rs

```diff
@@ -1170,14 +1170,13 @@ impl ToplevelSurface {
     ///
     /// If the parent is `None`, the parent-child relationship is removed.
     pub fn set_parent(&self, parent: Option<wl_surface::WlSurface>) -> bool {
```

Comment:
WlSurface -> &WlSurface would probably make this nicer to use.

Reply:
yeah, done.